cmd/destroy/remote.go

```diff
@@ -117,19 +123,24 @@ type remoteDestroyCommand struct {
 
 func newRemoteDestroyer(manifest *model.Manifest, ioCtrl *io.Controller) *remoteDestroyCommand {
 	fs := afero.NewOsFs()
-	builder := remoteBuild.NewBuilderFromScratch(ioCtrl)
+	runner := &buildCmd.OktetoBuilder{
```

Comment:
NIT: Could we move this into a init function? I think right now is not a problem but if we ever end up adding more fields to buildCmd.OktetoBuilder, we can forget to add that field on all the initializations. WDYT?

Reply:
I created the init function but that doesn't prevent you from that error. You can always create the OktetoBuilder outside the init function for whatever reason